charts/agent/templates/daemonset.yaml

```diff
@@ -219,7 +219,7 @@ spec:
               value: /opt/draios/etc/ca-certs/{{- include "sysdig.custom_ca.keyName"  (dict "global" .Values.global.ssl "component" .Values.ssl) -}}
             {{- end }}
           readinessProbe:
-            {{- if ge (semver "12.18.0" | (semver .Values.image.tag).Compare) 0 }}
+            {{- if eq (include "agent.enableHttpProbes" .) "true" }}
             httpGet:
               path: /healthz
```

Comment:
A totally unrelated comment, is the `z` at the end on purpose?

Reply:
Yes, it's a sort of convention